doc/Overview/Logging/LoggingOverview.md

```diff
@@ -1,108 +1,174 @@
 ---
 uid: Overview.Logging
 ---
+
 # Logging
-Uno.Extensions.Logging use [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) for logging abstractions.
 
-For more documentation on logging, read the references listed at the bottom.
+Apps that record events typically do so for informational or diagnostic purposes. **Logging** is the process of recording events to either a _persistent_ store such as a text file or database, or a _transient_ location like the standard output console. The `Uno.Extensions.Logging` library leverages logging capabilities tailored to the target platform to easily record events for XAML layout, Uno-internal messages, and custom events. It allows for control over specific severity or verbosity levels. This feature also allows for a simple way to wire up custom log providers. It uses [Microsoft.Extensions.Logging](https://www.nuget.org/packages/Microsoft.Extensions.Logging) for logging abstractions.
 
+For more documentation about logging, read the references listed at the bottom.
 
 ## Platform Log Providers
 
-Wire up platform specific log providers (iOS: Uno.Extensions.Logging.OSLogLoggerProvider, WASM: Uno.Extensions.Logging.WebAssembly.WebAssemblyConsoleLoggerProvider), Debug and Console logging.
+To wire-up the platform specific log providers for debug and console logging, use the extension method `UseLogging()` on the `IHostBuilder` instance:
 
 ```csharp
-private IHost Host { get; }
+protected override void OnLaunched(LaunchActivatedEventArgs e)
+{
+    var appBuilder = this.CreateBuilder(args)
+        .Configure(host => {
+            host.UseLogging();
+        });
+...
+```
+
+## Logging
 
-public App()
+When the `UseLogging()` extension method is called, an `ILogger<T>` is registered in the service provider. This logger can be used to log messages to the console. It can then be injected into any class that needs to log messages. The `ILogger<T>` interface is generic, where `T` is the class that is logging the message. While the `ILogger<T>` interface is generic, it is not required to be used in this way. It can be used without the generic type parameter, but it is recommended to use the generic type parameter to scope the logger to a specific class.
```

Comment:
You need to be careful with "While the ILogger<T> is generic, it is not required to be used this way" <-- when you retrieve from the service provider it **must** specify T. Trying to resolve ILogger will not work